bin/git-fork

```diff
@@ -17,9 +17,13 @@ read -r user
 # personal access token
 # config name is github-personal-access-token '_' is not allowed in git config
 
-github_personal_access_token=$(git config git-extras.github-personal-access-token)
+if test -n "${GITHUB_TOKEN}"; then
+  github_personal_access_token="$GITHUB_TOKEN"
+else
+  github_personal_access_token=$(git config git-extras.github-personal-access-token)
```

Comment:
```suggestion
  github_personal_access_token=$(git config --default '' git-extras.github-personal-access-token)
```

Would recommend passing `--default` to make code more robust. (in the future, `errexit` may be enabled, and this invocation shouldn't exit the entire script if the config key could not be found).